src/lib/isMobilePhone.js

```diff
@@ -96,6 +96,7 @@ const phones = {
   'pl-PL': /^(\+?48)? ?[5-8]\d ?\d{3} ?\d{2} ?\d{2}$/,
   'pt-BR': /^((\+?55\ ?[1-9]{2}\ ?)|(\+?55\ ?\([1-9]{2}\)\ ?)|(0[1-9]{2}\ ?)|(\([1-9]{2}\)\ ?)|([1-9]{2}\ ?))((\d{4}\-?\d{4})|(9[2-9]{1}\d{3}\-?\d{4}))$/,
   'pt-PT': /^(\+?351)?9[1236]\d{7}$/,
+  'pt-AO': /^(\+244)\d{9}$/,
```

Comment:
Here are the changes I am requesting: 

1. Within Angola, `911123432` is a valid phone number. 
2. Also, we should allow `244911123432` as valid. 
3. Update the ReadMe under `isMobilePhone()` with `pt-AO`

Please do the changes above and update your tests. 
Looking forward to updates.

Reply:
Also remember to update the list in the README.

Reply:
@profnandaa i will update.
@ezkemboi 
there is no need to validate "911123432" this is only valid in Angola. The need to validate +244911123432. All platforms use this way. Nobody uses 911123432. I'm Angolan.